tests/coprocessor/test_select.rs

```diff
@@ -893,6 +893,37 @@ fn test_order_by_column() {
 }
 
 #[test]
+fn test_order_by_pk() {
+    let mut data = vec![
+        (8, Some("name:0"), 2),
+        (7, Some("name:3"), 3),
+        (6, Some("name:0"), 1),
+        (5, Some("name:6"), 4),
+        (4, Some("name:5"), 4),
+        (3, Some("name:4"), 4),
+        (2, None, 4),
+    ];
+
+
+    let product = ProductTable::new();
+    let (_, mut end_point) = init_with_data(&product, &data);
+    let req = Select::from(&product.table)
+        .order_by(product.id, true)
```

Comment:
I don't think this case is related to this fix. If you remove the fix, can this case pass?